### DISCUSSION TITLE
Vimeo Player Stuck Layer Loading

### DISCUSSION THREAD

**Discussion by rodox12** [vimeo/player.js#1019] [Bug reports]
Expected Behavior
Video plays using the Javascript's API

Actual Behavior
Video plays but the loading layer doesn't disappear, it keeps showing an endless circle animation above the video.
It happens randomly.

I am using Chrome and tested in 2 different PCs and already disabled the QUIC Protocol.

![Screenshot from 2023-08-15 10-08-05](https://github.com/vimeo/player.js/assets/19394214/35a2f6c9-5bb9-4cd2-9f00-a504b7a5ee5c)

**Comment by rkrishnan8594** [comment]
We'll take a look at this issue when there are steps for reproduction. For what it's worth, it seems unlikely that the Player.js library would be the cause behind re-buffering issues, so I'd encourage you to confirm that being testing whether the issue still occurs with the library removed.
